lib/steps.js

```diff
@@ -573,6 +573,44 @@ Then(/the "([^"]+)" link opens in (a new window|the same window)/, async (linkTe
   await scope.afterStep(scope);
 });
 
+Then(/(?:the )?text(?: in)?(?: the)?(?: JSON)?(?: var)?(?:iable)?(?: )?"([^"]+)" should be/i, async (var_name, expected_value) => {
+  /* Check that the text string of the page's JSON var value matches
+  *    the given text string. Does not currently accept nested values
+  *    like "foo.bar". The word "text" is required.
+  *
+  * The format for this step is something like this:
+  * ```
+  * Then the text in the JSON variable "var_name" should be
+  * """
+  * A single or
+  * multiline string
+  * """
+  * ```
+  *
+  * Sentence combinations that will work:
+  * `the text in the JSON variable "variable_with_text_value" should be`
+  * `the text in the JSON var "variable_with_text_value" should be`
+  * `the text in "variable_with_text_value" should be`
+  * `text var "variable_with_text_value" should be`
+  */
+
+  // Get JSON variable value
+  let { filename, json } = await scope.savePageJSONData( scope );
+  let actual_value = json.variables[ var_name ];
+
+  // Add warning to report if value is not as expected
+  if ( actual_value !== expected_value ) {
+    await scope.addToReport( scope, {
+      type: `error`,
+      value: `The variable "${ var_name }" was not equal to the expected `
+        + `value on the "${ scope.page_id }" screen. Check the downloads `
+        + `file ${ filename } to see the page's JSON variables.`
+    });
+  }
+
+  expect( actual_value ).to.equal( expected_value );
```

Comment:
Does the `expect` here print the actual value and the expected value of the variable? In my experience, those values are much more helpful than just the name of the variable, and it currently looks like the tester has to go to the saved JSON report file and to the original test that failed to see what those values are.

Reply:
It does print the values.